app/controllers/session_controller.rb

```diff
@@ -15,6 +15,7 @@ class SessionController < ApplicationController
   allow_in_staff_writes_only_mode :email_login
 
   ACTIVATE_USER_KEY = "activate_user"
+  FORGOT_PASSWORD_EMAIL_LIMIT_PER_DAY = 6
```

Comment:
this limit is way way tighter than old limit ... (6 an hour / 3 a minute) ... we should not be fiddling with this and just keep old limit for now.

Reply:
This should be the old limit and all I did was extract the constant out from https://github.com/discourse/discourse/pull/27314/files#diff-b955d5e07bb336431ce5d25acd60fe8f87e3a31c4b33a167a069f3b76bf3ef64L625.

Reply:
Yeah seems the same to me as well?

Reply:
Oh so many limits ... we also have these: 

https://github.com/discourse/discourse/blob/62b1b69e820bfa48898d5f654c2b71162078a73a/app/controllers/session_controller.rb#L610-L612

I guess this is OK then...

Reply:
And 

https://github.com/discourse/discourse/blob/eebf3320252061176a41d1807c4289e359059418/app/controllers/session_controller.rb#L815-L829

as well which is called in a `before_action` callback for the `SessionController#create` method. 

In theory, we might not even hit the RateLimiter for sending out the forgot password email because an earlier RateLimiter will be triggered first. That being said, I think it is OK even if we aren't super efficient with our RateLimiter.

Reply:
Yes TGX I forgot to say I actually hit the login rate limiter before I hit the reset password one in some cases. IMO this is fine too, it's not something that will commonly happen.